v1/bundle/store_test.go

```diff
@@ -2258,11 +2358,6 @@ func TestBundleLazyModeLifecycleRawNoBundleRoots(t *testing.T) {
 				},
 				"etag": "foo"
 			}
-		},
-		"modules":{
-			"example/example.rego":{
-				"rego_version":1
-			}
```

Comment:
No explicit rego-version set for runtime and bundle, so both default to the same version (`v1`) and is therefore not recorded into the stored meta data.